conan/internal/api/install/generators.py

```diff
@@ -80,6 +81,9 @@ def write_generators(conanfile, app, envs_generation=None):
     global_generators = load_cache_generators(HomePaths(app.cache_folder).custom_generators_path)
     hook_manager.execute("pre_generate", conanfile=conanfile)
 
+    # Generate sbom
+    _generate_sbom(conanfile)
```

Comment:
rename to something like ``generate_graph_manifests``?

Reply:
If this is an user plugin, it might need to have precedence over recipe-generated files?
in that sense, it might need to run after the other generators not before?